velox/functions/prestosql/coverage/Coverage.cpp

```diff
@@ -301,15 +320,32 @@ void printVeloxFunctions() {
 
   auto scalarNames = getSortedScalarNames();
   auto aggNames = getSortedAggregateNames();
-
-  const int columnSize = std::max(maxLength(scalarNames), maxLength(aggNames)) +
+  auto windowNames = getSortedWindowNames();
+
+  // Remove aggregate functions from window functions.
```

Comment:
This works, but it quite inefficient (which probably doesn't matter here). Consider moving this logic into getSortedWindowNames and use exec::aggregateFunctions() map to filter out window functions.

```
 for (const auto& entry : functions) {
    if (aggregateFunctions.count(entry.first) == 0) {
       names.push_back(entry.first);
    }
  }
```